common/build/eslint-config-fluid/src/custom-rules/no-member-release-tags.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const { TSDocParser } = require("@microsoft/tsdoc");
+
+/**
+ *
+ * @param comment: String - The comment included inside the class / interface / type.
+ * @returns hasReleaseTag: Boolean - True; if the comment contains any release tags.
```

Comment:
The consumers of the API won't know or should care what the variable was called inside the function.
```suggestion
 * @returns `true` if the comment contains any release tags; `false` otherwise.
```